CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Keep it human-readable, your future self will thank you!
 - Feat: Anemoi Profiler compatible with mlflow and using Pytorch (Kineto) Profiler for memory report [38](https://github.com/ecmwf/anemoi-training/pull/38/)
 - New limited area config file added, limited_area.yaml. [#134](https://github.com/ecmwf/anemoi-training/pull/134/)
 - New stretched grid config added, stretched_grid.yaml [#133](https://github.com/ecmwf/anemoi-training/pull/133)
+- Add reader groups to reduce CPU memory usage and increase dataloader throughput [#76](https://github.com/ecmwf/anemoi-training/pull/76)
```

Comment:
Due to changelog bot fun, this is in the wrong space, could you put it in the unrelease bit please. (Sorry for the chore)

Reply:
done: [05665c7](https://github.com/ecmwf/anemoi-training/pull/76/commits/05665c7f8257bf75092049f4f6a71de4055b2fc7)